### PR TITLE
fix: Support to read Hive table without partition key

### PR DESCRIPTION
**Pull request by zhaojintaozhao**
Fix issue302, support to read Hive table without partition key

### CODE REVIEW

**Comment by zhaojintaozhao** [comment]
> Fix issue302, support to read Hive table without partition key

This PR may fix issue #302

File: src/Storages/Hive/HiveDataPart.h [hunk 1]
```diff
@@ -71,25 +71,32 @@ struct HivePartInfo
...
-        std::vector<String> values;
-        boost::split(values, temp, boost::is_any_of("/"), boost::token_compress_on);
-
-        for(auto elem : values)
+        else
         {
-            std::vector<String> key_value;
-            boost::split(key_value, elem, boost::is_any_of("="), boost::token_compress_on);
-            partition.insert({key_value[0], key_value[1]});
+            partition.insert({"", ""});
```
Comment (Clark0) [inline src/Storages/Hive/HiveDataPart.h:99]: can leave `partition` empty if there's no partition

**Comment by hustnn** [comment]
@zhaojintaozhao The build failed, pls take a look.

**Comment by zhaojintaozhao** [comment]
> @zhaojintaozhao The build failed, pls take a look.

Yes, I fix this build failed issue.

File: src/Storages/Hive/HiveDataPart.h (hunk 1 above)
Comment (zhaojintaozhao) [reply src/Storages/Hive/HiveDataPart.h]: I resolve this conversation

**Review by wordhardqi — APPROVED** [review]

**Comment by WillemJiang** [comment]
@zhaojintaozhao  Could you take a look at the build failure after resolving the merge conflict?

**Review by Clark0 — APPROVED** [review]

**Pull request merged**